### ISSUE TITLE
Cannot connect via web

### ISSUE THREAD

**Issue by Verguldebarman** [subparpedestrian/cphw-web#1]
First of all, thanks for your work!!

I tried both web & android and unfortunately, I cannot get it working. For Android, I've posted a [separate issue ](https://github.com/subparpedestrian/cphw-android/issues/1)on your git. I'm sharing below my issue with the web version.

What I have done and what works:

1. I have downloaded the code from github
2. unzipped the files in cphw-web-main folder
3. started index.html in cphw-web-main folder
4. wheel website starts with status: disconnected and all buttons
5. only [connect] button is clickable, the rest is "greyed out"
6. when I click [connect] a pop-up is shown to connect with BT
7. I can see my CW and also I can select in by clicking [pair]
8. the pop-up disappears and I am returned to the website

and now the issue:

- status remains on status: disconnected no matter what I do...
- on the wheel all five LEDs keep blinking every second (from the manual I think this means "on/locked")

Troubleshooting:

- used different laptops
- used different browsers (chrome/edge)

Any clue?

**Comment by subparpedestrian** [comment]
Hello,
You can go here if you want to skip downloading the source: https://subparpedestrian.github.io/cphw-web/ it is automatically updated with the latest commits.
Are you familiar with Chrome/Edge developer console (F12)?  If so, look at the console and see if there are any error messages reported.
Another person has used some of my source and created another project you can see if that works for you: [https://tb47.me/chw/lite.html](https://tb47.me/chw/lite.html)

**Comment by Verguldebarman** [comment]
I've made a video to show what I mean with the LEDs. [Here](https://github.com/subparpedestrian/cphw-web/assets/67055019/684f01f8-19d7-464b-947c-f49ab3303b4b
)

I've also tried the [other project](https://tb47.me/chw/lite.html), and it looks like it does a bit more. I'll make a video tomorrow. But the end is the same, no electric support while driving.

I need to mention that I live in the Netherlands, so I have a European Wheel, 250W. No clue if that makes any difference. 

The last time the wheel has seen some action was in 2022, so long time ago...  Since then the account was locked, so couldn't login.

**Comment by subparpedestrian** [comment]
Interesting.  Does the battery % in the webapp change when the LEDs change?  Also note that you will not get assist when the wheel is upside down.  I can try to add some more diagnostic output (battery voltage, present hazards) soon, but quite busy this week.

**Comment by Verguldebarman** [comment]
Got the wheel working with this project; https://tb47.me/chw. 

Although currently only from my laptop and not (yet) from my Android phone, but that will be possible after some troubleshooting. Thanks for your quick replies.

**Comment by Verguldebarman** [comment]
> Interesting. Does the battery % in the webapp change when the LEDs change? Also note that you will not get assist when the wheel is upside down. I can try to add some more diagnostic output (battery voltage, present hazards) soon, but quite busy this week.

Yes, the battery % in the webapp when connected. Also see https://github.com/subparpedestrian/cphw-web/issues/2 as I can connect now, but have  a new issue with e-braking. Also and still the eco, standard and turbo buttons are not functional for me.
